cmd/discovery/cmd/send.go

```diff
@@ -32,18 +32,37 @@ This application is a tool to generate the needed files
 to quickly create a Cobra application.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println("send called")
-		fmt.Println(*id)
 
-		discovery.SendPing(*myport, *targetid, *targetport)
+		nodeId, err := discovery.NewNodeFromString(*bootstrapNode)
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+
+		myAddr, err := net.ResolveUDPAddr("udp", *addr)
```

Comment:
"udp" is case-sensitive?
and better to define it as constant.

Reply:
this is a golang buildin method, I don't think we need to move it to constant